spec/policies/task_policy_spec.rb

```diff
@@ -24,20 +24,20 @@
   describe '#create?' do
     subject { policy.create? }
 
-    let(:creator) { User.new(id: 43) }
-    let(:project) { Project.new(id: 42, user: creator) }
-    let(:task) { Task.new(project: project) }
+    let(:project) { create :project, user: creator }
+    let(:task) { create :task, project: project, user: user }
+    let(:user) { create :user }
 
     context 'when user is creator of the project' do
-      let(:user) { creator }
+      let(:creator) { user }
```

Comment:
Maybe it is more correct to code it like
```suggestion
      let(:user) { creator }
```
?